.circleci/config.yml

```diff
@@ -1100,6 +1139,7 @@ workflows:
       - t_ems_compile_ext_colony: *workflow_emscripten
       - t_ems_compile_ext_gnosis: *workflow_emscripten
       - t_ems_compile_ext_zeppelin: *workflow_emscripten
+      - t_ems_compile_ext_ens: *workflow_emscripten
```

Comment:
Maybe we also need to add the test jobs here? Wonder why it wasn't added in the first place.

Reply:
Exactly my thoughts. I thought I didn't read it right the first time :sweat_smile:

Reply:
Created an issue to track this https://github.com/ethereum/solidity/issues/10397